build/config/InsertionSignToolData.json

```diff
@@ -1,14 +1,37 @@
 {
-    "sign": [
-      {
-        "certificate": "VsixSHA2",
-        "strongName": null,
-        "values": [
-          "insertion\\Microsoft.FSharp.SDK.Resources.ENU.vsix"
-        ]
-      }
-    ],
-    "exclude": [
-    ]
-  }
-  
\ No newline at end of file
+  "sign": [
+    {
+      "certificate": "VsixSHA2",
+      "strongName": null,
+      "values": [
+        "insertion\\Microsoft.FSharp.Compiler.vsix",
+        "insertion\\Microsoft.FSharp.Compiler.Resources.*.vsix",
+        "insertion\\Microsoft.FSharp.SDK.Resources.*.vsix"
+      ]
+    }
+  ],
+  "exclude": [
+    "fsc.exe",
+    "FSharp.Build.dll",
+    "FSharp.Build.resources.dll",
+    "FSharp.Core.dll",
+    "FSharp.Core.resources.dll",
+    "FSharp.Compiler.Private.dll",
+    "FSharp.Compiler.Private.resources.dll",
+    "FSharp.Compiler.Server.Shared.dll",
+    "FSharp.Compiler.Interactive.Settings.dll",
+    "FSharp.Compiler.Interactive.Settings.resources.dll",
+    "fsi.exe",
+    "fsiAnyCpu.exe",
+    "FSharp.Data.TypeProviders.dll",
```

Comment:
Why are the above excluded - that doesn't seem right.

Reply:
The listed files have either already been signed during this build (fsi, etc.) or come from a nuget package (microsoft.build.*).

Reply:
Can it be commented somehow so that people know why these are excluded?

Reply:
Sadly, no.  The JSON file format specification doesn't allow comments as per [RFC 7159](http://www.ietf.org/rfc/rfc7159.txt).